v2/tools/asoctl/cmd/root.go

```diff
@@ -32,7 +35,8 @@ func newRootCommand() (*cobra.Command, error) {
 	rootCmd.Flags().SortFlags = false
 
 	cmdFuncs := []func() (*cobra.Command, error){
-		NewExportCommand,
+		export.NewExportCommand,
```

Comment:
minor: Given these are in packages we probably could just call it `NewCommand`? If we've got a package per top-level command anyway?

Then we get:
```
export.NewCommand(),
crd.NewCommand()
```

which maybe reads 25% prettier?